### DISCUSSION TITLE
Case mounting tabs and CT numbers

### DISCUSSION THREAD

**Discussion by presslab-us** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#33] [Show and tell]
I wanted to use the Orbit 57095 Timer Cabinet to house the energy meter: https://www.orbitonline.com/product/gray-outdoor-timer-cabinet/

The provided 3D printed case didn't really have mounting like I wanted it, so I have modified the STL files to add mounting tabs that line up with existing holes in that cabinet.  This cabinet will fit up to the 24-input (3 addon board) energy meter.

The second change was adding numbers to the sides of the case so it can be quickly seen what the CT input number is.

Ideally these changes would be in the source CAD file, and not just the STLs, so if this improvement is desired that would be the proper place to make these changes.  I didn't see the CAD files available on GitHub.

Here's the OpenSCAD remix and STLs for a 6-input and 24-input version: [case_mod.zip](https://github.com/CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter/files/4672481/case_mod.zip)

![IMG_20200523_164225](https://user-images.githubusercontent.com/5122115/82740244-fd221f00-9d14-11ea-8624-345095ca7b70.jpg)
![IMG_20200523_164322](https://user-images.githubusercontent.com/5122115/82740247-ff847900-9d14-11ea-9cab-0cc768a4d441.jpg)

**Comment by CircuitSetup** [comment]
Thanks for sharing your case! I'll have the tabs in the master. Also, very clean install!

**Comment by mbeaudoin** [comment]
It would be nice to have access to the CAD files for the case as well, not just the STL files. Then we could mod it as necessary.

**Comment by ICPete** [comment]
I second that request for 3D case files. Format could be Solidworks, STEP, or Parasolid. A set of part files for just one configuration (for example, for two add-on boards) would be adequate, as it could be easily modified within Solidworks or other CAD software. In my case, I'm strongly considering replacing all the 3.5mm mini-phone jacks with plug-in screw terminal blocks (Amphenol Anytek OQ021251000G & TJ0211530000G).  Rather than having to file out all the round holes to rectangular holes to accommodate these new connectors, I'd love to be able to revise the housing design myself. While at it, I might also add the jack numbering, and definitely add some mounting feet, per presslab-us' modifications above. BTW, I bought boards etc from you to build two systems: one with two add-on boards (18 channels total) and one with four add-on boards (30 channels total); I'll be installing these at two different load panels in different buildings.

**Reply by graveland** [reply]
The store page for the case has a link for the fusion 360 project at https://a360.co/2NwbN9q

Store page: https://circuitsetup.us/product/expandable-6-channel-energy-meter-3d-printed-case/

**Reply by icvdok** [reply]
Hello graveland, may ask to share your f360 project? Your link return:

Unauthorized
Sorry. You don't have permission to access this page.

Many thanks

**Comment by ICPete** [comment]
Got it, thanks! I got the free hobbyist version of Fusion and was able to export the model to STEP format. All set!
